backend/src/config/opencast.rs

```diff
@@ -68,6 +72,10 @@ impl OpencastConfig {
         self.upload_node.as_ref().unwrap_or_else(|| self.unwrap_host())
     }
 
+    pub(crate) fn external_api_node(&self) -> &HttpHost {
+        self.external_api_node.as_ref().unwrap_or_else(|| self.unwrap_host())
+    }
+
```

Comment:
This is the third instance of this pattern, at which point I personally would want a thing for it. 😅